modules/booklist.js

```diff
@@ -0,0 +1,15 @@
+export default class BookList {
+  constructor() {
+    this.books = [];
+  }
+
+  checkBooks() {
+    if (localStorage.getItem('books')) {
+      this.books = JSON.parse(localStorage.getItem('books'));
+    }
+  }
+
+  saveBooks() {
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
```

Comment:
- [ ] As mentioned in the Project requirements we have to use ES6 syntax **"Refactor your methods and functions to arrow functions."** please update all your functions to use arrow functions syntax.

Reply:
done